src/content/configuration/node.md

```diff
@@ -9,29 +9,18 @@ contributors:
   - byzyk
   - EugeneHlushko
   - anikethsaha
+  - chenxsan
 ---
 
-The following Node.js options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
+The following Node.js options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html).
```

Comment:
`node` only cares about three Node.js globals now, so informations related to `modules` are removed.

Reply:
`This allows code originally written for the Node.js environment to run in other environments like the browser.` was removed as it's not true anymore, we have to polyfill Node.js modules like `fs` etc too, which is described in `resolve.fallback` section.